components/header/header.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from 'react';
+import style from './header.module.scss';
```

Comment:
I've been trying to put the style import as the last thing at the top of each file, [like here](https://github.com/kedro-org/kedro-website/blob/main/components/why-kedro/why-kedro.tsx), to try and separate concerns a little bit at the top (e.g. all JS imports and then styles below that).